src/js/order.js

```diff
@@ -138,34 +158,62 @@ function extractDetailFromDoc(order, doc) {
         return a;
     };
     const us_tax = function(){
-        const a = extraction.by_regex(
-            [
-                '//div[contains(@id,"od-subtotals")]//' +
-                'span[contains(text(),"tax") ' +
-                'and not(contains(text(),"before") ' +
-                ')]/' +
-                'parent::div/following-sibling::div/span',
-
-                '//*[text()[contains(.,"tax") and not(contains(.,"before"))]]',
-
-                '//div[contains(@class,"a-row pmts-summary-preview-single-item-amount")]//' +
-                'span[contains(text(),"tax")]/' +
-                'parent::div/following-sibling::div/span',
-
-                // Example: 'Tax Collected: $0.77'
-                '//div[@id="digitalOrderSummaryContainer"]//*[text()[contains(., "Tax Collected: ")]]',
-
-                '//*[contains(text(), "tax to be collected")]/parent::*/following-sibling::*/descendant::*/text()'
-            ],
-            /(?:vat:|tax:|tax collected:)? *((?:GBP |USD |CAD |EUR |AUD)?[$£€]?-?[.0-9]+)/i,
-            '9.99',
-            doc.documentElement
-        );
-        if (a) {
-            return a;
+        var a
+        try {
+            a = extraction.by_regex(
+                [
+                    './/tr[contains(td,"Tax Collected:")]',
+                ],
+                null,
+                'n/a',
+                doc.documentElement
+                );
+            a = a.match('Tax Collected:\\s+(((?:GBP|USD|CAD|EUR|AUD)?)\\s?(([$£€]?)\\s?(\\d+[.,]\\d\\d)))')[1];
+        } catch (_) { 
+            try {
```

Comment:
I think that multiply nested try-catches aren't great (I'm sure I've done it - e.g. I am a hypocrite).
Could we instead work through an array of options, returning N/A if none of the work.
A slightly annoying niggle in coding this is that the obvious "forEach" doesn't work with a return in the lambda you pass in, so you have to either use a sentinel value for failure (like null), or use an exception to signal success. I think that explicitly iterating using a for() might be cleaner.

Reply:
You'll probably need to explain this. Meanwhile, I got rid of the try/catch for now.

Reply:
I meant that code like 
try {
     a();
} catch ex1 {
     b();
     ...
} catch ex2 {
    c();
    ...
} catch ...

seems to be heading the direction of diminishing returns instead of
function tryInTurn(options){
   // try each option in turn and return the first one that works (for some definition of work - either no-throw or returning something "good")
}

const wantedResult = tryInTurn([a,b,c,d]);
// with a following wind, tryInTurn can be multiply reused.

No need to adopt this immediately btw...